core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -840,7 +840,9 @@ class HttpGoogleServicesDAO(
     import spray.json.DefaultJsonProtocol._
     import DeploymentManagerJsonSupport._
 
-    val templateLabels = parseTemplateLocation(dmTemplatePath).map(_.toJson).getOrElse(Map("template_path" -> labelSafeString(dmTemplatePath)).toJson)
+    val templateLabels = parseTemplateLocation(dmTemplatePath).map(_.toJson).getOrElse(Map("template_path" -> labelSafeString(dmTemplatePath)).toJson).convertTo[Map[String, String]]
+
+    val projectLocationLabel = Map("project_location" -> location.get) //todo
```

Comment:
`.get` will go away once i make location non-optional. shouldn't be a problem for now as all fields were backfilled